files/default/neutron-ha-tool.py

```diff
@@ -73,7 +73,8 @@ def make_argparser():
     ap.add_argument('--l3-agent-migrate', action='store_true', default=False,
                     help='Migrate routers away from offline l3 agents')
     ap.add_argument('--l3-agent-evacuate', default=None, metavar='HOST',
-                    help='Migrate routers away from a particular l3 agent')
+                    help='Migrate routers away from l3 agent living on HOST, '
+                    'where HOST is the name of the host')
```

Comment:
Uhm... I feel the new description a bit repetitive. Is clear that HOST is the name of the host. But if you think that it helps, OK for me.